package/src/store/SqliteClient.ts

```diff
@@ -1,5 +1,5 @@
-import type { DB, OPSQLite } from '@op-engineering/op-sqlite';
-let sqlite: typeof OPSQLite;
+import type { DB, OPSQLiteProxy } from '@op-engineering/op-sqlite';
```

Comment:
@khushal87 I know we talked about this before, why didn't we move to this type specifically ?

Reply:
This was because the underlying types were broken as a result after the change and hence we decided not to touch it right now.

Reply:
Ah, right - I remember. It seems like the proxy is working fine type-wise now (both Typescript doesn't complain and we're getting autocomplete and stuff); so I think it's fine like this. Do you agree ?